lua/autorun/server/sv_cfc_weapon_lockouts_init.lua

```diff
@@ -0,0 +1,17 @@
+local function addFiles( dir )
+    local files, dirs = file.Find( dir .. "/*", "LUA" )
+    if not files then return end
+    for k, v in pairs( files ) do
+        if string.match( v, "^.+%.lua$" ) then
+            AddCSLuaFile( dir .. "/" .. v )
+        end
+    end
+    for k, v in pairs( dirs ) do
+        addFiles( dir .. "/" .. v )
+    end
+end
```

Comment:
In a project where new files won't be frequently added or without many files, I find these kinds of solutions to be overkill, but if you think it fits then I'll let it be

Reply:
Gotcha, I just use this method to quickly set up client and shared files really quickly so I don't have to bother messing around with file paths and making sure I got everything. Really handy, especially when I'm working on setting up a bunch of these mini-addons together like today.